### PR TITLE
Fix python versions on travis

### PR DESCRIPTION
**Pull request by MaxNoe**
Travis was not actually testing different python versions, because we install anaconda and don't make sure which python version was installed using anaconda, so it was always what came with miniconda.

Currently 3.7.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,45 @@
...
+    - name: "python=3.7, ctapipe=0.7"
+      python: 3.7
+      env:
+        - CTAPIPE_VERSION="v0.7.0"
+        - CTAPIPE_IO_LST_VERSION="v0.4.1"
+
+    - name: "python=3.6, ctapipe=master"
+      python: 3.6
+      env:
+        - CTAPIPE_VERSION="master"
```
Comment (rlopezcoto) [inline .travis.yml:28]: do we really want to test against ctapipe `master`?

File: environment.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ channels:
   - cta-observatory
   - conda-forge
 dependencies:
+  - python=3.7
```
Comment (rlopezcoto) [inline environment.yml:7]: why would we be testing against python 3.6 and requiring 3.7?

Comment (MaxNoe) [reply environment.yml:7]: Again, what is in this environment is not what you require. It is an environment specification.
You can only have a single version of python in an environment.

Which python version you require is set in setup.py (`python_requires` I think), and ctapipe currently supports `>=3.6`, so I thought here it is the same.

File: .travis.yml (hunk 1 above)
Comment (MaxNoe) [reply .travis.yml:28]: This was what was there before. I think it is good. It is under allowed failures.

But this gives a nice heads up, that as soon as we are compatible to the next release, when we diverged from master again.

Comment (rlopezcoto) [reply .travis.yml:28]: ok, but it will start failing very soon after we are even with ctapipe master

**Pull request merged**